docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,534 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fe59d027-121a-4411-9dc6-0c1756f205e2",
+   "metadata": {},
+   "source": [
+    "# Alert activations\n",
```

Comment:
@jillnogold actually that's what Tomer suggested and I think it's a good idea to begin with what alert activations are and then how to list them. So I don't think this should be fixed

Reply:
@rokatyy  The problem is that the heading of the topic "Alert activations" does not really describe what is inside. It could be viewing, creating, deleting, whatever you imagine. 
How about naming the page "Viewing alert activations" and keeping the listing heading. Then all the subheadings (listing, filtering and grouping) fit underneath as different ways to "view" the activations.
On a different track - how long are the records stored? Is there a max to the storage? Auto-delete? Manual delete? What does the user need to know?

Reply:
@jillnogold moved this part to alerts.md and added info about partitioning and retention.